app/_locales/ar/messages.json

```diff
@@ -70,14 +84,6 @@
     "message": "MetaMask",
     "description": "The name of the application"
   },
-  "appNameBeta": {
```

Comment:
These two keys are also used in the manifest, and need to be preserved across locales.

Reply:
updated in the crowdin portal to avoid these deletions and now re-running the action.

Reply:
seems to have worked. These are no longer getting deleted  🙌

Reply:
Great, thanks!